src/librustc_llvm/ffi.rs

```diff
@@ -288,6 +288,9 @@ pub enum RelocMode {
     Static = 1,
     PIC = 2,
     DynamicNoPic = 3,
+    ROPI = 4,
+    RWPI = 5,
+    ROPI_RWPI = 6,
```

Comment:
OK, do you think it makes more sense to add numbers in the C++ or remove them here? (the other C++ enums don't have them, so I'm assuming remove them here)